hooks/analytics/useAPYChart.ts

```diff
@@ -1,21 +1,46 @@
-import { useQuery } from "react-query";
-import { useMemo, useState } from "react";
+import { useMemo, useState, useEffect } from "react";
 import {
   borderFormatting,
   filterByDuration,
   formatDisplay,
 } from "../../utils/analytics";
+import { fetchApyHistory } from "../../utils";
 
 export const useAPYChart = () => {
-  const { data, isFetching } = useQuery(`/api/analytics/charts/apy`, {
-    initialData: {
-      labels: [],
-      datasets: [],
-      error: null,
-    },
-    refetchOnWindowFocus: false,
-    keepPreviousData: true,
+  const [isFetching, setIsFetching] = useState(true);
+  const [data, setData] = useState({
+    labels: [],
+    datasets: [],
+    error: null,
   });
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const json = await fetchApyHistory(360);
```

Comment:
no way or reason to refine how many days we pull?

Reply:
Not sure I understand... the 360 arg is the number of days to pull?

Reply:
i guess this chart is always just the same # of days? was wondering why we don't have the number of days in a param or something